docs/root/intro/version_history.rst

```diff
@@ -141,6 +141,7 @@ Version history
 * redis: added :ref:`success and error stats <config_network_filters_redis_proxy_per_command_stats>` for commands.
 * redis: migrate hash function for host selection to `MurmurHash2 <https://sites.google.com/site/murmurhash>`_ from std::hash. MurmurHash2 is compatible with std::hash in GNU libstdc++ 3.4.20 or above. This is typically the case when compiled on Linux and not macOS.
 * redis: added :ref:`latency_in_micros <envoy_api_field_config.filter.network.redis_proxy.v2.RedisProxy.latency_in_micros>` to specify the redis commands stats time unit in microseconds.
+* retry: added :ref:`envoy.retry_host_predicates.omit_canary_hosts <envoy_api_field_route.RetryPolicy.retry_host_predicate>` retry predicate
```

Comment:
can you break this up into two lines? try to keep each line below 100 chars wide

Reply:
Is this a good way to put it?

```rst
* retry: added a retry predicate that :ref:`rejects canary hosts. <envoy_api_field_route.RetryPolicy.retry_host_predicate>`
```
I think removing the full name of the predicate is okay as the link to the doc is right there.

Reply:
Yeah that sounds good to me.